controller-server/src/test/java/com/yahoo/vespa/hosted/controller/restapi/user/UserApiTest.java

```diff
@@ -120,6 +125,11 @@ public void testUserManagement() {
                                       .roles(Set.of(roles.tenantOperator(id.tenant()))),
                               new File("application-roles.json"));
 
+        // GET application role information is available also under the /api prefix.
+        tester.assertResponse(request("api//user/v1/tenant/my-tenant/application/my-app")
```

Comment:
did you intend to do api//user or should this be /api/user?

Reply:
What, how could this pass?

Reply:
Some normalisation is probably done, then. No, I didn't intend for this.

Reply:
So this was interpreted as just `/user` — the `8080api` port was interpreted as port 8080. Nice.